### PR TITLE
Terry's overlay

### PR DESCRIPTION
**Pull request by kprevas**
Adds HTML and CSS for Terry's initial overlay design.  Syncing of game state to Firebase is in a JS file which should be usable across designs (may require a few tweaks to selectors).

### CODE REVIEW

File: gamestate.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+function fillHand(handElt, deck) {
+    handElt.empty();
+    for (var i = 0; i < deck.length; i++) {
+        var card = deck[i];
+        if (card.inhand) {
+            var cardElt = $("<div>", {"class": "card"});
+            cardElt.text(card.name);
+            var costElt = $("<span>", {"class": "mana"});
+            costElt.html(toCost(card.cost && ("" + card.cost)));
```
Comment (cdanis) [inline gamestate.js]: are you a wizard

File: gamestate.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            cardElt.append(costElt);
+            handElt.append(cardElt)
```
Comment (cdanis) [inline gamestate.js]: honestly asking: does JS care about missing semicolons?

File: gamestate.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    $(".sideboard .left").text(v.val().sideboard.replace(/_/g, "\n"));
+});
+firebase.database().ref('p1deck').on('value', function (v) {
+    fillHand($("#p1hand"), v.val());
+});
+firebase.database().ref('player2').on('value', function (v) {
+    console.log(v.val());
+    $(".right .life").text(v.val().life);
+    var poison = v.val().poison;
+    var poisonElt = $(".right .poison");
```
Comment (cdanis) [inline gamestate.js]: so when you did this were you just following my conventions or did what I did actually make sense

File: gamestate.js (hunk 1 above)
Comment (kprevas) [inline gamestate.js]: lol javascript

File: gamestate.js (hunk 2 above)
Comment (kprevas) [inline gamestate.js]: It doesn't, except when it does!

File: gamestate.js (hunk 3 above)
Comment (kprevas) [inline gamestate.js]: I was following your conventions but it makes sense to me

**Pull request merged**